test/test_images/runtime/handlers/cgroup.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Knative Authors
```

Comment:
The license is formatted here differently. And below.
Not sure if this matters, but looks non-uniform.